test/new-e2e/tests/process/testing.go

```diff
@@ -34,10 +36,14 @@ var systemProbeConfigStr string
 //go:embed config/npm.yaml
 var systemProbeNPMConfigStr string
 
+//go:embed compose/fake-process-compose.yaml
+var fakeProcessCompose string
+
 // assertRunningChecks asserts that the given process agent checks are running on the given VM
-func assertRunningChecks(t *assert.CollectT, vm *components.RemoteHost, checks []string, withSystemProbe bool, command string) {
+func assertRunningChecks(t *assert.CollectT, client agentclient.Agent, checks []string, withSystemProbe bool) {
+	status := client.Status(agentclient.WithArgs([]string{"--json"}))
+	assert.NotNil(t, status, "failed to get agent status")
```

Comment:
Updates the status check to use the `agentclient.Agent` interface which is environment agnostic (ex. host vs docker).